test/demo/utils/uuidv4.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Generates a random uuid based on math.random and the system time
+ */
+export function uuidv4() {
+    return (Math.random().toString(36).substring(2) + Date.now().toString(36));
+}
```

Comment:
I get nervous when I see something claim it's uuidv4 and it doesn't use proper uuid code. Tell me in a convincing way that this is OK, or consider having a dev dependency on uuid?

Reply:
Used a different uuidv4 function.